theme/templates/resource-landing-page/resource-header.html

```diff
@@ -18,12 +18,13 @@
                     </td>
 
                     <td v-if="res_mode === 'Edit'">
+                        {% if not cm.raccess.published %}
                         <a :disabled="isUpdatingAuthor" type="button" id="btn-add-new-author"
```

Comment:
I understand you're just following the prior conventions, but if it's not a large effort to consolidate towards using vue instead of the Django template conditionals throughout, then that's ideal. The content model variable cm should be available in this scope. Also, there are global JS variables available to you from the parents.

Reply:
I am aware of the fact that using vue would cleanup lot of mess both in template and JS code. However, unfortunately I lack any experience in vue to be able to use vue to fix these issues now.